Assets/VRTK/Scripts/Presence/VRTK_BodyPhysics.cs

```diff
@@ -1035,13 +1038,15 @@ protected virtual void StartFall(GameObject targetFloor)
             isMoving = false;
             isLeaning = false;
             onGround = false;
+            fallMinTime = Time.time + (Time.fixedDeltaTime * 3.0f); // Wait at least 3 fixed update frames before declaring falling finished 
```

Comment:
Why the 3 frames?

3 feels a bit like a magic number. I'm guessing this is caused by a timing issue and 3 happens to work? Wonder if there's a better way of solving the timing issue. @bddckr any ideas?